vumi/components/message_store.py

```diff
@@ -1102,3 +1119,22 @@ def key_with_ts_and_value_formatter(batch_id, result):
 def key_with_rts_and_value_formatter(batch_id, result):
     key, reverse_ts, value = key_with_ts_and_value_formatter(batch_id, result)
     return (key, from_reverse_timestamp(reverse_ts), value)
+
+
+@inlineCallbacks
+def add_batches_to_event(stored_event):
+    """
+    Post-migrate function to be used with `vumi_model_migrator` to add batches
+    to stored events that don't have any.
+    """
+    if stored_event.batches.keys():
+        # We already have batches, so there's no need to look them up.
+        returnValue(False)
```

Comment:
Its a bit of a small edge case, but isn't it possible that we'd still want to re-add batches in case the migration crashed half-way through migrating a message?